docs/api-reference/deckgl.md

```diff
@@ -75,9 +75,13 @@ This is helpful when rendered objects are difficult to target, for example
 irregularly shaped icons, small moving circles or interaction by touch.
 Default `0`.
 
-##### `pixelRatio` (Number, optional)
```

Comment:
OK. 3.0 was the first release we promoted to the public so that should be OK to just remove it then. I suspect are no apps that use 2.0.